database/drop_customer.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS customers;
```

Comment:
can we combine all drop commands into one sql file?

Reply:
I got no opinion on this, i'm fine either way. Maybe check with @liawsy and @hellopanda128

Reply:
Same I don’t think it makes a huge difference but combining probably makes it more readable?

Reply:
i think combining all the drop commands would be neater :)

Reply:
Combined drop scripts into one